src/Workspaces/SharedUtilitiesAndExtensions/Workspace/Core/LanguageServices/AddImports/AddImportHelpers.cs

```diff
@@ -39,25 +41,56 @@ internal static class AddImportHelpers
                 var newFirstUsing = newImports[0].WithLeadingTrivia(
                     firstToken.LeadingTrivia.Where(t => !IsDocCommentOrElastic(syntaxFacts, t)));
                 newImports[0] = newFirstUsing;
+
+                for (var i = 0; i < newImports.Count; i++)
+                {
+                    var trailingTrivia = newImports[i].GetTrailingTrivia();
+                    if (!syntaxFacts.IsEndOfLineTrivia(trailingTrivia.Count == 0 ? default : trailingTrivia[^1]))
```

Comment:
```suggestion
                    if (trailingTrivia.Count == 0 || !syntaxFacts.IsEndOfLineTrivia(trailingTrivia[^1]))
```

seems easier to understand for me :)

Reply:
➡️ Simplified this case along with the existing location where this pattern was taken from